server/athenian/api/internal/account.py

```diff
@@ -432,31 +430,20 @@ async def copy_teams_as_needed(
     if not team_rows:
         log.warning("Found 0 metadata teams for account %d", account)
         return [], 0
-    # check for cycles - who knows?
-    dig = {}
-    for row in team_rows:
-        team_id = row[MetadataTeam.id.name]
-        if (parent_id := row[MetadataTeam.parent_team_id.name]) is not None:
-            dig[team_id] = [parent_id]
-        else:
-            dig[team_id] = []
+    # TODO: avoid cyclic import
+    from athenian.api.internal.team import get_meta_teams_members, get_meta_teams_topological_order
```

Comment:
No, we shouldn't do this in production code. Let's factor the common functions to a separate module.